### PR TITLE
misc: Remove code duplication

### PR DESCRIPTION
**Pull request by ZJvandeWeg**
## Checklist

 - [ ] I have read the [contribution guidelines](https://github.com/FlowFuse/flowfuse/blob/main/CONTRIBUTING.md)
 - [ ] I have considered the performance impact of these changes
 - [ ] Suitable unit/system level tests have been added and they pass
 - [ ] Documentation has been updated

### CODE REVIEW

File: .eleventy.js [hunk 1]
```diff
@@ -312,17 +312,17 @@ module.exports = function(eleventyConfig) {
         }
     });
 
-    eleventyConfig.addShortcode("renderTeamMember", function(teamMember) {
+    eleventyConfig.addShortcode("renderTeamMember", function (teamMember) {
```
Comment (ZJvandeWeg) [inline .eleventy.js:315]: Just formatting updates, no semantical updates

File: src/_includes/layouts/webinar.njk [hunk 2]
```diff
@@ -31,38 +31,34 @@ layout: layouts/base.njk
...
-                    <div class="team-card--sm">
-                        <div class="ff-headshot" style="background-image: url(/images/team/headshot-{{ team[host].headshot }})"></div>
-                        <div class="team-card-info">
-                            <label>{{ team[host].name }}</label>
-                            <span>{{ team[host].title }}</span>
-                        </div>
-                    </div>
+                        {% if team[host] %}
+                            {% renderTeamMember people[host] %}
+                        {% endif %}
```
Comment (ZJvandeWeg) [inline src/_includes/layouts/webinar.njk:52]: @Yndira-FlowForge This is the only true change

File: .eleventy.js [hunk 3]
```diff
@@ -312,17 +312,17 @@ module.exports = function(eleventyConfig) {
...
         return `<div class="team-card--sm">
-                    <div class="ff-headshot" style="background-image: url(/images/team/headshot-${ teamMember.headshot })"></div>
+                    <div class="ff-headshot" style="background-image: url(/images/team/headshot-${teamMember.headshot})"></div>
                         <div class="team-card-info">
-                            <label>${ teamMember.name }</label>
-                            <span>${ teamMember.title }</span>
+                            <label>${teamMember.name}</label>
+                            <span>${teamMember.title}</span>
                         </div>
                     </div>`
```
Comment (ZJvandeWeg) [inline .eleventy.js:327]: @Yndira-FlowForge Is this missing a `</div>`?
```suggestion
                    </div>`
```

Comment (Yndira-FlowForge) [reply .eleventy.js:327]: It isn't, but the indent isn't right. From line 323 onward, all lines have an extra space to the right.

**Review by Yndira-FlowForge — APPROVED** [review]

**Pull request merged**